libs/build_dod_dataset.py

```diff
@@ -43,16 +43,49 @@ def get_abbrev_df():
 def get_projections_3_26_df():
     return pd.read_csv('projections_03-26-2020.csv')
 
-def get_hospitals_and_shortfalls(projection, days_out):
-    for row in projection:
-        row_time = datetime.datetime.strptime(row[DATE], '%m/%d/%y')
+all_cols = [
+    "day_num",
+    # ^ for index column
+    "date",
+    "a",
+    "b",
+    "c",
+    "d",
+    "e",
+    "f",
+    "g",
+    "all_hospitalized",
+    "all_infected",
+    "dead",
+    "beds",
+    "i",
+    "j",
+    "k",
+    "l",
+    "population",
+    "m",
+    "n",
+]
+
+exclude_cols = ["a","b","c","d","e","f","g","i","j","k","l","m","n"]
+
+def read_json_as_df(path):
+    df = pd.DataFrame.from_records(simplejson.load(open(path,'r')),
```

Comment:
What's the reason for simplejson vs json here?

Reply:
it's faster for w/e reason